### PR TITLE
check php version early

### PR DESCRIPTION
**Pull request by remicollet**
See http://forum.glpi-project.org/viewtopic.php?pid=339918#p339918

### CODE REVIEW

**Comment by remicollet** [comment]
Of course, GLPI_MIN_PHP cannot be used as not yet defined.

**Comment by trasher** [comment]
What about creating a new file with only the `defines` ad load it at the very begining?
Having to change the minimal PHP version at two different place may land to mistakes in the future :/

**Comment by remicollet** [comment]
From phpcompatinfo static analyser report on index.php

`Requires PHP 5.3.0 (min), PHP 5.3.0 (all)`

**Comment by remicollet** [comment]
> What about creating a new file with only the defines ad load it at the very begining?

To be honest, having 1 more file only for this seems messy
More, all the other checks have no value (never reached code), only the one here is useful.

**Comment by trasher** [comment]
OK then. But there are chances this one will not be changed next time PHP version will be bumped :D

File: inc/define.php [hunk 1]
```diff
@@ -47,7 +47,7 @@
    //for stable version
    define("GLPI_SCHEMA_VERSION", '9.2');
 }
-define('GLPI_MIN_PHP', '5.6.0');
+define('GLPI_MIN_PHP', '5.6.0'); // Must also be changed in top of index.php
```
Comment (trasher) [inline inc/define.php]: 👍

**Pull request merged**